app/views/categories/index.html.erb

```diff
@@ -0,0 +1,39 @@
+<section class="categories-container">
+  <div class="categories-header">
+      <div class ="exit">
+      <%= button_to 'Sign out', destroy_user_session_path, method: :delete, class: 'delete-btn' %>
+       </div>
+      <h2>CATEGORIES</h2>
+      <div class ="search">
+        <%= image_tag "search.png", alt: "Search Icon", class: "search-img"%>
+      </div>
+  </div>
+  
+  <ul class="categories">
+ 
+  <% @categories.each do |category| %>
+    <% if user_signed_in? && category.user_id == current_user.id %>
+     <li class="category-container">
+     <div class="cat-icon">
+     <%= image_tag category.icon, alt: "category Icon", class: "cat-image"%>
+     </div>
+        <div class="cat-info-cont">
+            <h3>
+                <%= link_to category.name, category_path(category), { class: "category-name" } %>
+            </h3>
+            <hr class="space">
+            <p class="category-updated">
+                 <%= category.updated_at.strftime('%d %b %Y')%>
+             </p>  
+        </div>
+            <p class="cat-amount">$<%= category.payments.sum(:amount) %></p>
+            
+     </li>
+ <% end %>
+ <% end %>
+</ul>
+<div class="new-cat-container">
+ <%= link_to "ADD CATEGORY", new_category_path, class: "new-category" %>
+</div>
```

Comment:
- [ ] Kindly try as much as possible to match the design given, this includes colors, space between elements, and typography. I have added images for your reference. Please pay attention to the colors that I have marked with red.

## Given design
![colors](https://user-images.githubusercontent.com/53124548/236610821-e2d40298-6370-4fb7-ac4b-45e399072735.png)

## Your design
![your-design](https://user-images.githubusercontent.com/53124548/236610884-81876071-cb36-48cb-ae11-63df7aa07664.png)